apps/public-docsite-v9/src/Concepts/Theming.stories.mdx

```diff
@@ -0,0 +1,140 @@
+
+<Meta title="Concepts/Developer/Theming" />
+
+# Theming
+
+The Fluent UI Theme is represented by a set of tokens. Each token resolves to a single value which can be assigned to a CSS property.
+
+You can browse all the available tokens in **Theme** section of the docs.
+
+## How theme is applied
+
+No matter what theme is used, the component styles are always the same. The only way to change the component styling is through theme tokens which can be used in style values.
+
+Those tokens are resolved to CSS variables. The `FluentProvider` component is responsible for setting the values of the CSS variables in DOM and changing them when the theme changes. When the theme is switched, only the variables are changed, all styles remain the same.
+
+Place a `<FluentProvider />` at the root of your app and pass a theme to the `theme` prop. The provider will render a `div` and set all tokens as CSS variables on that element.
+
+```jsx
+import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
+
+export const AppRoot = ({children}) => (
+  <FluentProvider theme={teamsLightTheme}>
+    {children}
+  </FluentProvider>,
+);
+```
+
+Import `tokens` to style a component using `makeStyles`
+
+```jsx
+import { tokens } from '@fluentui/react-theme';
+
+const useStyles = makeStyles({
+  root: { display: 'flex' },
+  rootPrimary: { color: tokens.colorNeutralForeground3 },
+});
+
+export Component = props => {
+  const classes = useStyles();
+
+  return <div className={mergeClasses('ui-component', classes.root, props.primary && classes.rootPrimary)} />;
+}
+```
+
+For more details, see [Styling components](?path=/docs/concepts-developer-styling-components--page).
+
+### Do not use CSS variables directly
+
+**⚠ Never use theme CSS variables directly!** The CSS variables implementation of the theme is internal to the library. We might eventually decide to change the variable names, hash them or even use direct values instead of some variables. Always use the `tokens` to access the theme.
+
+## Available themes
+
+Fluent UI currently exports following themes:
+
+- Web Light (`webLightTheme`)
+- Web Dark (`webDarkTheme`)
+- Web High Contrast (`webHighContrastTheme`)
+- Teams Light (`teamsLightTheme`)
+- Teams Dark (`teamsDarkTheme`)
+- Teams High Contrast (`teamsHighContrastTheme`)
+
+### High contrast themes
+
+⚠ Do not use High Contrast themes! All Fluent UI components support Windows High Contrast mode automatically regardless of the active theme.
+Windows high contrast mode is the recommended high contrast platform for all customers using Fluent UI.
+
+Hardcoded High Contrast themes are considered legacy, to be used only in applications which explicitly support those.
+
+## Customizing the theme
+
+Applications can customize a theme in multiple ways.
+
+### Custom Brand ramp
+
+The brand ramp is a color ramp going from dark to light colors:
+
+<img src={require('../../public/brand-ramp-example.png')} alt="Example of a brand ramp" />
```

Comment:
Can we use a real component instead? We have a component for this ramp, so it will be always up to date.

Reply:
Neither the `Ramp` component nor the brand ramp itself is exported, leaving as screenshot for now.